data/json/monsters.json

```diff
@@ -3070,13 +3070,34 @@
         {
             "type" : "bite",
             "cooldown" : 10,
-            "damage_max_instance" : {
-                "type" : "stab",
-                "amount" : 10,
-                "armor_multiplier" : 0.2
-            }
+            "damage_max_instance" : [
+                { "damage_type" : "stab", "amount" : 10, "armor_multiplier" : 0.2 }
+            ]
         },
-        ["IMPALE", 25]
```

Comment:
The ideas a whole isn't bad but this makes the JSON considerably more difficult to read. Implementing very complex behaviors in code is probably preferable to this syntax.